src/Nri/Colors.elm

```diff
@@ -172,32 +172,32 @@ blue =
 
 {-|
 
-<p style="font-size:2em; color: #00488A">#00488A</p>
+<p style="font-size:2em; color: #0e5595">#0e5595</p>
 
 -}
 blueDark : Css.Color
 blueDark =
-    hex "#00488A"
+    hex "#0e5595"
```

Comment:
@avh4 Alrighty, I just pushed changes so that the old color names point to the new names. Since the descriptions should be going in the style guide, should I get rid of them completely or is this PR good to go?